_posts/2023-08-02-#27827.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: pr
+date: 2023-08-02
+title: "[silent Payments]: Base functionality"
+pr: 27827
+authors: [josibake]
+components: ["wallet"]
+host: josibake
+status: upcoming
+commit:
+---
+
+## Notes
+
+This PR implements [BIP352: Silent Payments](https://github.com/bitcoin/bips/pull/1458) in the Bitcoin Core wallet. BIP352 is "a protocol for static payment addresses in Bitcoin without on-chain linkability of payments or a need for on-chain notifications."
+
+Before reviewing the PR, it is _strongly_ recommended that you read the [BIP](https://github.com/bitcoin/bips/pull/1458), as well as the [reference implementation](https://github.com/bitcoin/bips/blob/cfe0771a0408a2d2de278d4e95bb9a33bd1615b2/bip-0352/reference.py) and [test vectors](https://github.com/bitcoin/bips/blob/cfe0771a0408a2d2de278d4e95bb9a33bd1615b2/bip-0352/send_and_receive_test_vectors.json). This PR implements the base functionality: sending and receiving. Additional features and broader RPC support are left for follow up PRs. The PR is heavily based on a proof of concept PR, so this PR review club will be primarily focused on:
+
+* Does this PR correctly implement the BIP?
```

Comment:
You can break this down into more specific functionality and add links to the code that does it. In the questions section you can ask "how is this implemented and is it correct" eg `EncodeSilentDestination`, `VerifySilentPayment`, detecting whether a transaction is a silent payment to us, calculating the keys from the address, etc.